<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EarthquakeDataExtractClass</class>
 <widget class="QWidget" name="EarthquakeDataExtractClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EarthquakeDataExtract</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1,9,1">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="l_reportPath">
       <property name="minimumSize">
        <size>
         <width>84</width>
         <height>34</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>84</width>
         <height>34</height>
        </size>
       </property>
       <property name="text">
        <string>Report路径</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_reportPath"/>
     </item>
     <item>
      <widget class="QPushButton" name="pb_setReportPath">
       <property name="text">
        <string>打开</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_txtPath">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>84</width>
         <height>34</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>84</width>
         <height>34</height>
        </size>
       </property>
       <property name="text">
        <string>txt路径:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_txtPath"/>
     </item>
     <item>
      <widget class="QPushButton" name="pb_setTxtPath">
       <property name="text">
        <string>打开</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="l_outputPath">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>104</width>
         <height>34</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>104</width>
         <height>34</height>
        </size>
       </property>
       <property name="text">
        <string>输出文件路径:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_outputPath"/>
     </item>
     <item>
      <widget class="QPushButton" name="pb_setOutputPath">
       <property name="text">
        <string>打开</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="l_dateInterval">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>日期区间;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_startDate"/>
     </item>
     <item>
      <widget class="QLabel" name="l_ganggang">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_endDate"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QToolButton" name="tb_settings">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>各种设置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tb_run">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>开始抽取</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>截取进度</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>异常:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tb_errorStatusButton">
       <property name="minimumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>没</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="EarthquakeDataExtract.qrc"/>
 </resources>
 <connections/>
</ui>
